tests/test_components_xgm.py

```diff
@@ -58,6 +59,8 @@ def test_xgm_getters(multi_xgm_run, source_sase):
     xgm.photon_energy_by_train()
     xgm.doocs_server()
     xgm.pulse_energy()
+    xgm.slow_train_energy()
+    assert isinstance(xgm.pulse_energy(series=True), pd.Series)
```

Comment:
We're not checking the actual data? 😇

Reply:
I didn't think it was necessary since we're basically just calling `DataArray.to_series()`, but I added tests for it anyway in 3ad2174.

Reply:
Thanks. I just try to not rely on implementation but interface for what needs testing 🙂